### PR TITLE
OPENTOK-47901: ringtone publisher start timeout error fixed

### PR DESCRIPTION
**Pull request by IGitGotIt**

### CODE REVIEW

**Review by jvalli — APPROVED** [review]
👍

File: Ringtones/Ringtones/ViewController.m [hunk 1]
```diff
@@ -36,10 +36,12 @@ @implementation ViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-    
-    _myAudioDevice = [[OTAudioDeviceRingtone alloc] init];
+
+    NSString* path = [[NSBundle mainBundle] pathForResource:@"bananaphone"
+                                                     ofType:@"mp3"];
+    _myAudioDevice = [[OTAudioDeviceRingtone alloc] initWithRingtone:[NSURL URLWithString:path]];
```
Comment (v-kpheng) [inline Ringtones/Ringtones/ViewController.m:42]: 🍌 ☎️ 
:-)

File: Ringtones/Ringtones/OTAudioDeviceRingtone.m [hunk 2]
```diff
@@ -54,18 +59,7 @@ - (void)playRingtoneFromURL:(NSURL*)url
...
-    // started running. Setting the category while the audio session is
-    // configured for voice chat (PlayAndRecord) will interrupt recording and
-    // cause problems if a publisher is running.
-    if (!self.isCapturing && !self.isRendering) {
-        AVAudioSession* audioSession = [AVAudioSession sharedInstance];
-        [audioSession setCategory:AVAudioSessionCategoryPlayback
-                            error:nil];
-        [audioSession setActive:YES error:nil];
-    }
-
```
Comment (v-kpheng) [inline Ringtones/Ringtones/OTAudioDeviceRingtone.m:68]: Just curious, what issues did this code introduce?  Sorry, I'm an iOS newb :-(

**Review by v-kpheng — APPROVED** [review]
LGTM! 💪 🚀

**Pull request merged**